packages/eslint-plugin/src/rules/await-thenable.ts

```diff
@@ -39,6 +43,8 @@ export default createRule<[], MessageId>({
       convertToOrdinaryFor: 'Convert to an ordinary `for...of` loop.',
       forAwaitOfNonAsyncIterable:
         'Unexpected `for await...of` of a value that is not async iterable.',
+      invalidPromiseAggregatorInput:
+        'Unexpected iterator of non-Promise (non-"Thenable") values passed to promise aggregator.',
```

Comment:
Technically, I think the wording should be like so?

```suggestion
        'Unexpected iterable of non-Promise (non-"Thenable") values passed to promise aggregator.',
```

Reply:
Yes! Thanks 👍